packages/code-du-travail-frontend/src/outils/common/Agreement/ShowAlert.tsx

```diff
@@ -1,58 +1,94 @@
-import { Alert, Paragraph, Text } from "@socialgouv/cdtn-ui";
+import { Alert, Paragraph, Text, theme } from "@socialgouv/cdtn-ui";
 import React from "react";
 
 import type { AgreementSupportInfo } from "./types";
+import { Agreement } from "../../../conventions/Search/api/type";
+import styled from "styled-components";
 
 type Props = {
-  currentIdcc: number;
+  currentIdcc: Agreement;
```

Comment:
On ne passe plus un idcc du coup :) 

```suggestion
  currentAgreement: Agreement;
```